### PR TITLE
fixed tool environment

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy [hunk 1]
```diff
@@ -18,6 +18,7 @@ package org.kie.jenkins.jobdsl
 
 class Constants {
     static final String MAVEN_VERSION = '3.8.1'
+    static final String MAVEN_TOOL = "kie-maven-${MAVEN_VERSION}"
     static final String UPSTREAM_BUILD_MAVEN_VERSION = '3.8.1'
     static final String JDK_VERSION = 'kie-jdk11'
```
Comment (Ginxo) [inline job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy:23]: shouldn't it be like
```suggestion
    static final String JDK_VERSION = '11'
    static final String JDK_TOOL = "kie-jdk${JDK_VERSION}"
```
in order to keep maven and java aligned? :thinking:

File: job-dsls/jobs/kie/main/kie_jenkinsScripts_PR.groovy [hunk 2]
```diff
@@ -2,8 +2,7 @@ import org.kie.jenkins.jobdsl.Constants
 
 // definition of parameters
 
-def javaToolEnv="KIE_JDK11_HOME"
-def mvnToolEnv="KIE_MAVEN_3_8_1_HOME"
+def mvnToolEnv=Constants.MAVEN_TOOL
 def repoBranch=Constants.BRANCH
 def organization=Constants.GITHUB_ORG_UNIT
 def javadk=Constants.JDK_VERSION
```
Comment (Ginxo) [inline job-dsls/jobs/kie/main/kie_jenkinsScripts_PR.groovy:8]: and then (based on the Constants file comment)
```suggestion
def javaToolEnv=Constants.JDK_TOOL
```

**Review by Ginxo — CHANGES_REQUESTED** [review]
wdyt about my proposal?

File: job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy (hunk 1 above)
Comment (mbiarnes) [reply job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy]: DONE.
All JAVA_VERSION were changed to JAVA_TOOL

File: job-dsls/jobs/kie/main/kie_jenkinsScripts_PR.groovy (hunk 2 above)
Comment (mbiarnes) [reply job-dsls/jobs/kie/main/kie_jenkinsScripts_PR.groovy]: DONE.
All JAVA_VERSION were changed to JAVA_TOOL

**Review by Ginxo — CHANGES_REQUESTED** [review]
please check, thanks for the effort

File: job-dsls/jobs/kie/kie-tools/deploy_development_version.groovy [hunk 3]
```diff
@@ -5,7 +5,7 @@ import org.kie.jenkins.jobdsl.Constants
 def nextDevVer="x.x.0-SNAPSHOT"
 def baseBranch=Constants.BRANCH
 def organization=Constants.GITHUB_ORG_UNIT
-def javadk=Constants.JDK_VERSION
+def javadk=Constants.JDK_TOOL
 def mvnVersion="kie-maven-" + Constants.MAVEN_VERSION
```
[the remark this answers is not in the document]
Comment (mbiarnes) [reply job-dsls/jobs/kie/kie-tools/deploy_development_version.groovy:9]: @Ginxo DONE

File: job-dsls/jobs/kie/kie-tools/kie_docker_ui_webapp.groovy [hunk 4]
```diff
@@ -12,7 +12,7 @@ def ghJenkinsfilePwd = "kie-ci"
 def repoBranch = Constants.BRANCH
 def gitHubJenkinsfileRepUrl = "https://github.com/${ghOrgUnit}/${repo}/"
 def mavenVersion = Constants.MAVEN_VERSION
```
[the remark this answers is not in the document]
Comment (mbiarnes) [reply job-dsls/jobs/kie/kie-tools/kie_docker_ui_webapp.groovy:14]: @Ginxo DONE

File: job-dsls/jobs/kie/kogito/kogito_docs_upload.groovy [hunk 5]
```diff
@@ -9,7 +9,7 @@ def currentKogitoDocsVersion = "0.17.0"
 def currentKogitoDocsTagName = "0.17.0-kogito"
 def nextKogitoDocsSnapshot = "0.18.0-SNAPSHOT"
 def sshKogitoDocsPath = "kogito@filemgmt.jboss.org:/docs_htdocs/kogito/release"
-def javadk=Constants.JDK_VERSION
+def javadk=Constants.JDK_TOOL
 def mvnVersion="kie-maven-" + Constants.MAVEN_VERSION
```
[the remark this answers is not in the document]
Comment (mbiarnes) [reply job-dsls/jobs/kie/kogito/kogito_docs_upload.groovy:13]: @Ginxo DONE

File: job-dsls/jobs/kie/main/automatic_web_publishing.groovy [hunk 6]
```diff
@@ -12,7 +12,7 @@ folder("KIE/${baseBranch}")
 folder("KIE/${baseBranch}/webs")
 def folderPath="KIE/${baseBranch}/webs"
 
-def javadk=Constants.JDK_VERSION
+def javadk=Constants.JDK_TOOL
 def mvnVersion="kie-maven-" + Constants.MAVEN_VERSION
```
[the remark this answers is not in the document]
Comment (mbiarnes) [reply job-dsls/jobs/kie/main/automatic_web_publishing.groovy:16]: @Ginxo DONE

File: job-dsls/jobs/kie/main/communityRelease_pipeline.groovy [hunk 7]
```diff
@@ -7,7 +7,7 @@ def organization=Constants.GITHUB_ORG_UNIT
 def m2Dir = Constants.LOCAL_MVN_REP
 def MAVEN_OPTS="-Xms1g -Xmx3g"
 def commitMsg="Upgraded version to "
-def javadk=Constants.JDK_VERSION
+def javadk=Constants.JDK_TOOL
 def mvnVersion="kie-maven-" + Constants.MAVEN_VERSION
```
[the remark this answers is not in the document]
Comment (mbiarnes) [reply job-dsls/jobs/kie/main/communityRelease_pipeline.groovy:11]: @Ginxo DONE

File: job-dsls/jobs/kie/main/dailyBuild_jdk11_pipeline.groovy [hunk 8]
```diff
@@ -1,6 +1,6 @@
 import org.kie.jenkins.jobdsl.Constants
 
-def javadk=Constants.JDK_VERSION
+def javadk=Constants.JDK_TOOL
 def mvnVersion="kie-maven-" + Constants.MAVEN_VERSION
```
[the remark this answers is not in the document]
Comment (mbiarnes) [reply job-dsls/jobs/kie/main/dailyBuild_jdk11_pipeline.groovy:4]: @Ginxo DONE

File: job-dsls/jobs/kie/main/dailyBuild_jdk11_prod_pipeline.groovy [hunk 9]
```diff
@@ -4,7 +4,7 @@ def baseBranch=Constants.BRANCH
 def organization=Constants.GITHUB_ORG_UNIT
 def kieVersion=Constants.KIE_PREFIX
 def m2Dir = Constants.LOCAL_MVN_REP
-def javadk=Constants.JDK_VERSION
+def javadk=Constants.JDK_TOOL
 def mvnVersion="kie-maven-" + Constants.MAVEN_VERSION
```
[the remark this answers is not in the document]
Comment (mbiarnes) [reply job-dsls/jobs/kie/main/dailyBuild_jdk11_prod_pipeline.groovy:8]: @ginxo DONE

File: job-dsls/jobs/kie/main/errai_pr.groovy [hunk 10]
```diff
@@ -17,7 +17,7 @@ def repoBranch = Constants.BRANCH
 def ghOrgUnit = "errai"
 def ghAuthTokenId = "kie-ci-token"
 def mvnGoals = "-B -e -fae -Dfull -Dmaven.test.failure.ignore=true -Pintegration-test clean install -Derrai.codegen.details=true -Dapt-generators"
-def javadk=Constants.JDK_VERSION
+def javadk=Constants.JDK_TOOL
```
[the remark this answers is not in the document]
Comment (mbiarnes) [reply job-dsls/jobs/kie/main/errai_pr.groovy:20]: @Ginxo DONE

File: job-dsls/jobs/kie/main/i18n_translations.groovy [hunk 11]
```diff
@@ -1,6 +1,6 @@
 import org.kie.jenkins.jobdsl.Constants
 
-def javadk=Constants.JDK_VERSION
+def javadk=Constants.JDK_TOOL
 def mvnVersion="kie-maven-" + Constants.MAVEN_VERSION
```
[the remark this answers is not in the document]
Comment (mbiarnes) [reply job-dsls/jobs/kie/main/i18n_translations.groovy:4]: @Ginxo DONE

File: job-dsls/jobs/kie/main/prodTag_pipeline.groovy [hunk 12]
```diff
@@ -12,7 +12,7 @@ def reportBranch=Constants.REPORT_BRANCH
 def MAVEN_OPTS="-Xms1g -Xmx3g"
 def cutOffDate = LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd"))
 def commitMsg="Upgraded version to "
-def javadk=Constants.JDK_VERSION
+def javadk=Constants.JDK_TOOL
 def mvnVersion="kie-maven-" + Constants.MAVEN_VERSION
```
[the remark this answers is not in the document]
Comment (mbiarnes) [reply job-dsls/jobs/kie/main/prodTag_pipeline.groovy:16]: @Ginxo DONE

**Comment by mbiarnes** [comment]
@Ginxo thanX for supervising this :)

**Review by Ginxo — APPROVED** [review]
LGTM

**Pull request merged**